frontend/src/pages/Landing/index.js

```diff
@@ -172,6 +178,26 @@ function Landing() {
       <UserContext.Consumer>
         {({ user }) => (
           <>
+            {showAlert && message && (
+            <Alert
+              type="success"
+              noIcon
+              cta={(
+                <Button
+                  role="button"
+                  unstyled
+                  aria-label="dissmiss alert"
+                  onClick={() => updateShowAlert(false)}
+                >
+                  <span className="fa-sm">
+                    <FontAwesomeIcon color="black" icon={faTimesCircle} />
+                  </span>
+                </Button>
+)}
```

Comment:
Need to match these brackets up with `cta` (I hate how eslint doesn't complain about this!)

Reply:
I was thinking of asking if ya'll would like to use something like [prettier](https://prettier.io/) to automatically format the code without thinking. nice to have everything consistent w/o worrying about it

Reply:
I would be all for it, maybe we should write a ticket for it in Jira?

Reply:
"format selection" can work here, no? But we could add prettier if you like it and it would make life easier (yes, something for Jira / post MVP).

Reply:
ah, I edit my code the old fashion way with just a plain text editor. So I don't get any of the goodies.